src/ui/components/NetworkMonitor/index.tsx

```diff
@@ -62,13 +66,16 @@ export const NetworkMonitor = ({
   }, [container.current]);
 
   if (loading) {
+    mixpanel.time_event("netMonitor.open_network_monitor");
     return (
       <div className="relative">
         <LoadingProgressBar />
       </div>
     );
   }
 
+  trackEvent("netMonitor.open_network_monitor");
```

Comment:
This one's fun — we'll be able to inspect this event and find out how long people are staring at the progress bar.

Reply:
Ah, this kind of answers my question above!

Reply:
Wait I want this everywhere now.